var/spack/repos/builtin/packages/macsio/package.py

```diff
@@ -21,7 +21,7 @@ class Macsio(CMakePackage):
     version("1.0", sha256="1dd0df28f9f31510329d5874c1519c745b5c6bec12e102cea3e9f4b05e5d3072")
 
     variant("mpi", default=True, description="Build MPI plugin")
-    variant("silo", default=True, description="Build with SILO plugin")
+    variant("silo", default=False, description="Build with SILO plugin")
```

Comment:
Unrelated?

Reply:
Yep, this was just to make Gitlab pass on an unrelated issue. Not sure why this lead to rebuilding `macsio`.